index.html

```diff
@@ -569,6 +576,9 @@ <h3><dfn>request()</dfn> method</h3>
             Let |promise:Promise| be
```

Comment:
Before this step, if |lock|.[[requestPromise]] is not undefined then should we return it rather than create a new Promise? This means we should also clear the internal slot when the Promise is resolved at the end of the parallel steps or rejected in the steps for `abort()`.

Reply:
yes that makes sense